documentation/asciidoc/topics/proc_install_command_line.adoc

```diff
@@ -40,7 +40,7 @@ EOF
 +
 [NOTE]
 ====
-You can omit `spec.targetNamespaces` from `OperatorGroup` to make {ispn_operator} available in all namespaces.
+Omit the entire `spec.targetNamespaces` block from `OperatorGroup` to make {ispn_operator} available in all namespaces.
```

Comment:
Maybe lets go from `spec.targetNamespaces` to `spec`? Otherwise if you would remove `targetNamespaces` you'll still get issue as you'll have empty `spec:` which isn't allowed. Therefore the need to remove the whole `spec` from our example

Reply:
That's a lot better. Thanks @Crumby !